packages/jellyfish-api-core/__tests__/category/masternode/unsetGov.test.ts

```diff
@@ -0,0 +1,73 @@
+import { RpcApiError } from '@defichain/jellyfish-api-core'
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { createToken } from '@defichain/testing'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+
+describe('Masternode', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+    await container.generate(20)
```

Comment:
Why is 20 blocks required to be generated? If there's a reason maybe you can add in comments to indicate as well.